do_checks.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 cargo fmt --check &&
-python .github/scripts/codecheck.py &&
+PYTHON=$(which python || which python3)
```

Comment:
Python never stops to piss me off... it seems I'll rewrite this script in rust and move on with my life.

Reply:
Xcode wanted to install python each time I tried to running non-existent `python`, and then failed installing it each time lol... so big hammer meets nail!